src/main.js

```diff
@@ -97,18 +117,25 @@ function favoriteIdea(target) {
         localStorage.setItem(id, JSON.stringify(idea));
       };
     };
-    showIdeas(savedIdeas)
+    showIdeas(savedIdeas.sort(compareId))
 };
 
 function pageLoad() {
   if (localStorage) {
     for (var i = 0; i < localStorage.length; i++){
       var key = localStorage.key(i);
       var idea = JSON.parse(localStorage.getItem(key));
-      var id = idea.id;
-      savedIdeas.push(new Idea(idea.title, idea.body, id, idea.star));
+      if (idea.title){
+        var id = idea.id;
+        savedIdeas.push(new Idea(idea.title, idea.body, id, idea.star));
+      }else if (!idea.title){
+        savedComments.push(new Comment(idea.id, idea.parentId, idea.text));
+      }
     }
-    showIdeas(savedIdeas);
+    for (var i = 0; i < savedComments.length; i++){
+      addCommentToIdea(savedComments[i], savedComments[i].parentId);
+    };
+    showIdeas(savedIdeas.sort(compareId));
```

Comment:
This seems like it works well - I could def use a review/explanation of the process from adding a comment until the end.